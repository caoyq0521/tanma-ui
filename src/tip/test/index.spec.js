import { mount } from '@vue/test-utils';
import TmTip from '../tip.vue';

describe('tmTip', () => {
  it('render tip', () => {
    const wrapper = mount(TmTip);
    expect(wrapper).toMatchSnapshot();
  });
})
  