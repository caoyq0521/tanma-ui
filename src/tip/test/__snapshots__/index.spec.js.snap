// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tmTip render tip 1`] = ``;
